### PR TITLE
Make `bundle exec rake serve` work out-of-the-box

### PR DESCRIPTION
**Pull request by Earlopain**
I wanted to do a simple styling fix but ended up having to deal with the javascript ecosystem :face_exhaling: 

There were a few things wrong, not sure why parcel especially made trouble. It's locked to a specific version in `package.lock`.

See individual commits for details. Just a warning, I don't know how this is deployed and if it is compatible with that. You can take this PR as a building block and fix things up yourself if you prefer.

### CODE REVIEW

File: app/assets/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import CodeMirror from 'codemirror/lib/codemirror'
```
Comment (marcandre) [inline app/assets/index.js]: For legibility, I'd keep `import 'codemirror/lib/codemirror'`. Or does that cause an issue?

**Comment by marcandre** [comment]
Sorry for your troubles, and thank you for the PR ❤️

File: app/assets/index.js (hunk 1 above)
Comment (Earlopain) [reply app/assets/index.js]: I don't think so, I guess my ide removed it since it is an unused import. I've added it back

**Comment by Earlopain** [comment]
It's not problem. Also, the fix I wanted to make isn't so easy afterall. I was bothered how highlighted text in the textarea doesnt change color when selecting it, making it hard to edit. I assume it has something to do with codemirror, the CSS I was trying to apply didn't take. So, I'll just keep living with that for now.

Would you mind some other PR bringing js dependencies up-to-date? I don't think I'll be doing codemirror 6, seems complex, but others probably are easier.

**Comment by marcandre** [comment]
> Would you mind some other PR bringing js dependencies up-to-date?

That would be nice, thank you

**Pull request merged**